package.json

```diff
@@ -43,10 +43,12 @@
     "next": "9.2.1",
     "node-fetch": "^2.6.0",
     "plottable": "^3.8.6",
+    "prop-types": "^15.7.2",
```

Comment:
Where are you using this? As far as I can tell, when using typescript, you don't need this.

Reply:
Removed unused package